README.md

```diff
@@ -89,14 +96,17 @@ The height and width of the image you wish to render.
 ##### `initialViewState` (object) [ONLY NECESSARY FOR `MicrsocopyViewer`]
 
 An object containing two things
+
 - `target` (Array) An `[x,y,0]` location in image coordinates of the image. The 0
   represents a hypothetical (and potentially future addition) of a third spatial dimension.
 - `zoom` (Number) The initial zoom level to render the image at
 
 ##### `minZoom` & `maxZoom` (Number)
 
 These control the max and min zoom sizes, generally the number of images `n` in your pyramid,
-ranging from `-n` (zoomed out) to `0`, the highest resolution.
+ranging from `-n` (`minZoom`), the lowest resolution, to `0` (`maxZoom`), the highest resolution. You don't need to
+pass these in (they can be inferred from heigh/width/tile size), although
+if you wish to limit zoom levels, you may.
```

Comment:
What happens if they are set?: Does it actually stop someone from zooming father out or in? When would someone want them?

Reply:
Yes, that is the use case.  We could not expose it - the `zarr` setting can infer it as well.